### PR TITLE
Feature/290 endless scroll

### PR DESCRIPTION
**Pull request by kkuepper**

### CODE REVIEW

File: components/EndlessDocumentList.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          console.error("error", e);
+        });
+    },
+    { distance: 10, interval: 500, canLoadMore: () => !fullyLoaded.value },
+  );
+});
+watch(
+  [useNuxtApp().$i18n.locale, () => contentLanguageStore().contentLanguages],
+  async () => {
+    const data = await props.load(0, position);
```
Comment (SimonSimCity) [inline components/EndlessDocumentList.vue:47]: We should also handle errors here... You do it well above, but here you leave errors to the watch-implementation to deal with.

File: components/EndlessDocumentList.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      }
+
+      loadingMore.value = true;
+      props
+        .load(position, 40)
+        .then((data) => {
+          position += 40;
+          if (data) {
+            if (data.length === 0) {
+              console.log(`list is fully loaded. position: ${position}`);
```
Comment (SimonSimCity) [inline components/EndlessDocumentList.vue:29]: Should we leave this console-log in?

**Review by SimonSimCity — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: components/EndlessDocumentList.vue (hunk 1 above)
Comment (kkuepper) [reply components/EndlessDocumentList.vue:44]: we're just swallowing the exception.
If it happens often, we should invest some time to figure out what's the best user experience.

**Pull request merged**